devito/propagator.py

```diff
@@ -308,7 +308,11 @@ def generate_loops(self, loop_body):
         omp_master = [cgen.Pragma("omp master")] if self.compiler.openmp else []
         omp_single = [cgen.Pragma("omp single")] if self.compiler.openmp else []
         omp_parallel = [cgen.Pragma("omp parallel")] if self.compiler.openmp else []
-        omp_for = [cgen.Pragma("omp for schedule(static)")] if self.compiler.openmp else []
+        reduction_list = ["kernel", "loop_body"] if self.compiler.openmp else []
+        omp_for = [cgen.Pragma("omp for schedule(static)"\
+                   + (self.profiler.get_loop_reduction("+", reduction_list)\
+                   if self.profile else ""))]\
+                  if self.compiler.openmp else []
```

Comment:
the + is only for flops, I did know that timing was not ok with openmp guess a little miscommunication here, I will add the change to handle loop timing